index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="Top">
+        <a href=""><img src="images/MyLogo.png" alt="My Logo" ></a>
+        <nav class="menubutton">
+            <button type="button"><img class="menu" src="images/menu.png" alt="Menu"></button>   
+        </nav>        
+    </header>
```

Comment:
It is a good habit to comment on your work so that third-party developers can understand what you are trying to accomplish